### PR TITLE
Non suppressed register

### PR DESCRIPTION
**Pull request by slubwama**
Revised Non Suppressed Register

### CODE REVIEW

**Comment by slubwama** [comment]
@ssmusoke You promised to review this PR. Just checking on status

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
             submitValStage.push(dateValidator("90267", "encounterDate", "", message4, "", "greater_than", true));
 
             //Validation for Start ART date
-            if (getValue("99161" + '.value') != "" &amp;&amp; getValue("99160" + '.value') == "") {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml]: @slubwama What has changed in this validation as the commit is not clear

**Comment by ssmusoke** [comment]
@slubwama  Feedback:

1. EMTCT and breastfeeding/pregnancy is not aligned, see screenshot screenshot 2018-05-08 17 14 00

2. VL Results at Initiation of IAC - should only show copies per ml, since a person only starts when they are unsupressed. See screenshot  screenshot 2018-05-08 17 15 45

3. Clinic Decision - add the codes and put Other Clinical decision in the same cell  

4. Regimen at VL Test, Regimen after IAC add the regimen codes 

5. Other Clinic decision is not aligned 
screenshot 2018-05-08 17 26 28

6. Outcome - add codes and consider putting Other Outcome in the same cell as Outcome codes

**Comment by slubwama** [comment]
@ssmusoke I have made some changes

**Comment by ssmusoke** [comment]
@slubwama We can make the regimens like MUAC

1. This is because there are forms where the regimens are written as letters so save the data entry clerk checking for the acutal regimen which can lead to mistake 

2. This is what we are pushing for consistently across all the dropdowns, radio buttons and checkboxes. as per the developer guide

**Comment by ssmusoke** [comment]
@slubwama see comment on Regimen codes which is also related to https://trello.com/c/vvxecFqv

**Comment by slubwama** [comment]
@ssmusoke basing on the email conversation, this wont be handled in this PR. So kindly request for final reviews and merging of this PR

**Comment by ssmusoke** [comment]
@slubwama 

1. The fields are still mis-aligned see attachments 
screenshot 2018-05-22 11 36 54
screenshot 2018-05-22 11 41 30
screenshot 2018-05-22 11 41 40
screenshot 2018-05-22 11 41 51

2. What are the required fields for this form which must be entered? Recommendations: 
- Date Results Received at Health Facility
- VL Results at Initiation 
- Date Results Given to Client 
- Regimen at VL Test 
- Start Date (should we not rename this to IAC Start Date) 

3. Put Regimen at VL Test in the same row at Date Results Received at Health Facility, since all that information is tied to VL test that was used for initiation 

4. Validation Rules:
- IAC session dates - should be after date of Registration
- Start Date should be on or after date of Registration 
- Date Results given to client must be after Date Results Received at the health facility 
- Date of Registration must be after Date Results Received at Health Facility 
- Date of VL Repeat must be after the latest date of IAC session 
- Date of Results Received (VL after IAC) must be after date of VL Repeat 
- Decision Date must be after Date Results Received (VL after IAC session)

**Comment by ssmusoke** [comment]
@slubwama 

1. I am using Firefox 

2. The agreed design for a viral load value is to have a date, qualitative value and actual viral load (when its detected)

3. #5 the two viral load values will override each other since the concepts are the same. 

4. The required fields are data that must be available on the paper tool before IAC. An alternative is to write data quality rules to flag those missing values, otherwise some indicators, like time from identification of non-supression to IAC start will not be available.

**Comment by slubwama** [comment]
@ssmusoke  I am using firefox too and those are the screen shots.

**Comment by slubwama** [comment]
@ssmusoke  I am going to Close this PR due to too many commits which cant squash

**Pull request closed without merging**